app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<table class="table table-striped custab">
+<thead>
+    <tr>
+        <th><%= t "model.categories.id" %></th>
+        <th><%= t "model.categories.name" %></th>
+        <th class="text-center"><%= t ".action" %></th>
+    </tr>
+</thead>
+    <% @categories.each do |category| %>
```

Comment:
em tìm hiểu về `render partial collection in rails ` đi
để khỏi phải foreach như thế này nhé